cogs/message_counter.py

```diff
@@ -49,7 +46,9 @@ async def on_message(self, message: commands.Context) -> None:
             messages_sent = messages_sent[0]
 
         # If the message was sent in a help channel, update the helpMessagesSent column
-        if "help" in category_name:
+        if (
+            category := getattr(message.channel, "category")
+        ) and "help" in category.name:
```

Comment:
Try to avoid complex code.
Walrus is unnecessary

Reply:
👍 Separated out in 467a68d7fadee8678645d01233c9dcaafb546d38